stable/traefik/README.md

```diff
@@ -122,8 +122,9 @@ The following table lists the configurable parameters of the Traefik chart and t
 | `acme.staging`                  | Whether to get certs from Let's Encrypt's staging environment        | `true`                                    |
 | `acme.logging`                  | Display debug log messages from the ACME client library              | `false`                                   |
 | `acme.domains.enabled`          | Enable certificate creation by default for specific domain           | `false`                                   |
-| `acme.domains.main`             | Main domain name of the generated certificate                        | `*.example.com`                           |
-| `acme.domains.sans`             | List of alternative subject name to give to the certificate          | `[]`                                      |
+| `acme.domains.domainList`       | List of domains & (optional) subject names                           | `[]`                                      |
```

Comment:
@billimek Does it suppose to be  `acme.domains.domainsList`? Missing `s` in the `Readme`